code-studio/src/js/videos.js

```diff
@@ -5,7 +5,9 @@ var videojs = require('video.js');
 var testImageAccess = require('./url_test');
 var clientState = require('./clientState');
 
-window.createVideoWithFallback = function (parentElement, options, width, height) {
+var videos = module.exports = {};
```

Comment:
Note: You could make this line

```javascript
var videos = module.exports;
```

Since `exports` is implicitly an empty object when browserify starts processing a module.